app/src/main/java/org/mozilla/focus/autocomplete/UrlAutoCompleteFilter.java

```diff
@@ -77,6 +77,12 @@ private void loadUrls(Context context) {
                 } catch (IOException e) {
                     // No autocomplete for you!
                     return null;
+                } finally {
+                    try {
+                        reader.close();
+                    } catch (IOException e) {
+                        // There's really nothing we can do now.
+                    }
```

Comment:
Reader implements Closable. So you should be able to use the helper in IOUtils (here and in the other cases). I wonder if our API level is high enough to actually use try-with-resources? That would be sweet.